roles/openshift_logging/files/curator.yml

```diff
@@ -0,0 +1,18 @@
+# Logging example curator config file
```

Comment:
We went with the approach that you can provide the contents of the file and the role will use what is provided or will fall back to the provided files as defaults